pystiche_papers/ulyanov_et_al_2016/_data.py

```diff
@@ -186,71 +185,74 @@ def images() -> DownloadableImageCollection:
     return DownloadableImageCollection({**content_images, **style_images})
 
 
+class SkipSmallIterableDataset(IterableDataset):
+    def __init__(
+        self,
+        dataset: Dataset,
+        min_size: int,
```

Comment:
Let's make everything but the dataset keyword only. This makes the instantiation more readable.

```suggestion
        dataset: Dataset,
        *
        min_size: int,
```